pkg/kubelet/apis/config/types.go

```diff
@@ -431,3 +434,11 @@ type SerializedNodeConfigSource struct {
 	// +optional
 	Source v1.NodeConfigSource
 }
+
+// LoggingConfig contains logging options
+type LoggingConfig struct {
```

Comment:
I expected this type to live in component config next to things like LeaderElectionConfiguration / DebuggingConfiguration / ClientConnectionConfiguration

Reply:
I proposed this location as LoggingConfig will be used for now by only Kubelet. I think there is no benefit of moving it to component config without helper functions like Validate, AddFlags etc. Until all flags are ready it's hard to coordinate work without hitting a lot of merge conficts.

 Migrating all flags into component config would be done as next step.

Reply:
It doesn't seem like we'd hit many merge conflicts over simple type additions. Since this is expected to be a common type, I think it makes sense to start it where we expect it to live.

Reply:
since we're adding an API type in this PR, let's add it in the place where we expect it to live (component-base)

define this type in staging/src/k8s.io/component-base/config/types.go, name it LoggingConfiguration, and reference it from kubelet/apis/config.KubeletConfiguration